frontend/src/components/QueueComponent/QueueTable/QueueTable.js

```diff
@@ -171,12 +171,20 @@ const QueueTable = observer(({ queueData }) => {
           {
             title: "Seen",
             //eslint-disable-next-line
+            /* eslint-disable */
```

Comment:
the single comment, then the multi-line comment is redundant
Please put each of these objects in this order:

```
title
sorting
align
//eslint-disable-next-line
render
```